Pinta.Gui.Widgets/Widgets/ColorPickerDialog.cs

```diff
@@ -867,61 +884,67 @@ private void DrawPaletteDisplay (Context g, Color c)
 
 	private void DrawColorSurface (Context g)
 	{
-		int draw_width = picker_surface_radius * 2;
-		int draw_height = picker_surface_radius * 2;
+		int radius = picker_surface_radius;
+		int doubleRadius = radius * 2;
+		Size drawSize = new (doubleRadius, doubleRadius);
 
-		ImageSurface img = CairoExtensions.CreateImageSurface (
+		using ImageSurface surface = CairoExtensions.CreateImageSurface (
 			Format.Argb32,
-			draw_width,
-			draw_height);
+			drawSize.Width,
+			drawSize.Height);
 
-		Span<ColorBgra> data = img.GetPixelData ();
+		Span<ColorBgra> data = surface.GetPixelData ();
 
-		if (picker_surface_type == ColorSurfaceType.HueAndSat) {
+		switch (picker_surface_type) {
 
-			int rad = picker_surface_radius;
+			case ColorSurfaceType.HueAndSat:
 
-			PointI center = new (rad, rad);
+				PointI center = new (radius, radius);
 
-			for (int y = 0; y < draw_height; y++) {
-				for (int x = 0; x < draw_width; x++) {
+				for (int y = 0; y < drawSize.Height; y++) {
+					for (int x = 0; x < drawSize.Height; x++) {
```

Comment:
This should be `drawSize.Width`, although the values are currently identical for both